### ISSUE TITLE
Preview is correct, but in editor (even as guest) the asset is rotated

### ISSUE THREAD

**Issue by braniii** [2d-inc/support#269]
I am facing a problem with rivr.app's editor. I have created an [animation](https://rive.app/a/braniii/files/flare/adonis/preview) which worked fine. After two days I opened it again and now a single asset is shifted (the red shirt sleeve). In the editor it says that it is still connected to the bone, but moving the bone does not have an effect on the shape. But, if one opens the file in the preview it works as intended.

In case this is not a bug and I made some stupid mistake I am sorry.

![Screenshot from 2020-04-30 21-38-46](https://user-images.githubusercontent.com/28560129/80752181-5be7e480-8b2b-11ea-970f-16049e2aca8d.jpg)

PS: And I just want to thank you for your tool and that your part of the open design movement.

**Comment by neurowave** [comment]
Thanks for reporting this, definitely a bug! I took a look at your file and found a quick workaround.

Select the path under your shirtR shape and delete the connected bone. Now hit cmd/ctrl z to undo. This fixes the issue.

![2020-04-30 13 39 11](https://user-images.githubusercontent.com/1817068/80757471-10ace200-8ae9-11ea-91a0-d02f97a07ca3.gif)

As a side note, I noticed that you're both connecting your shapes to bones and placing them under the bones in the hierarchy. This isn't necessary. If you're binding (connecting shapes to bones to deform vertices) then you don't need to create a hierarchical connection, the shape can even be on the root of your hierarchy. On the other hand, if you're just moving a shape with a single bone, then it's overkill to use binding… this takes up extra resources. It's more performant to just drop the shape under the bone in the hierarchy. In the case of this shirtR piece, it seems like you're only moving it with a  single bone and not deforming the vertices. This is the perfect case to use a hierarchical connection and not binding.

I realize this is all a bit confusing to read in text, so here are some videos we created on the topic that should help!

Connecting bones with hierarchical connections:
https://www.youtube.com/watch?v=A2wHCwPePFk

Connecting bones with binding (vector):
https://www.youtube.com/watch?v=jMyMuiQzxEo

Connecting bones with binding (raster):
https://www.youtube.com/watch?v=rADPLs36gQQ&t=1s
